upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -107,20 +108,128 @@ backup of Workload Manager configuration data and files is created. Once complet
+   options for configuring either the new CAN or CHN.
+
+Pivoting the pre-1.2 CAN to the new CMN allows administrative traffic (already on the pre-1.2 CAN) to remain as-is while
+moving standard user traffic to a new site-routable network.
+
+### UAN migration
+
+Certain steps are taken in order to minimize disruption to UANs during the CSM 1.2 upgrade process:
+
+1. During the upgrade, the switch `1.2 Preconfig` will not remove UAN ports from the CMN VLAN (the pre-1.2 CAN), allowing UANs
+   to retain their existing IPs during the CSM 1.2 upgrade process. Traffic to and from UANs will still flow through CMN, but
+   may also flow through CAN/CHN networks if desired.
+
+1. CFS will be temporarily disabled for UANs, so that running CFS plays does not remove CMN interfaces from UANs. As mentioned
+   in [Abstract (Stage 0.3)](#abstract-stage-03), network configuration is controlled by data in SLS, but CFS plays also pick up
+   the same SLS data, which can lead to UANs being prematurely removed from CMN and causing UAN outage. As such, CFS plays
+   need to be disabled for UANs.
+
+   To disable CFS plays for UANs, you must remove CFS assignment for UANs by running the following command:
+
+   ```bash
+   what-is-the-command-to-disable-CFS-for-UANs.sh
+   ```
+
+1. UAN reboots must be avoided. Rebooting a UAN can re-enable CFS and can ultimately lead to removing CMN interfaces from UANs.
+   As a system administrator, you must inform your users to avoid UAN reboots during the CSM 1.2 upgrade process. If, however,
+   a UAN is rebooted, you can run the following set of commands to add the CMN interface back on the UAN:
+
+   ```bash
+   what-is-the-command-to-readd-CMN-to-UANs-after-accidental-reboot.sh
+   ```
+
+1. Once the CSM 1.2 upgrade is complete, you may reboot the UANs for the new network configuration changes to take effect.
+   UANs will not receive an IP on the CMN network and instead will default their traffic through the new CAN/CHN.
+
+1. `{TODO: keepme?}` Note that in CSM 1.2, UAN ports will not be removed from the CMN VLAN7 in switches. In the next CSM release, switch
+   configuration will be updated to remove UAN ports from the CMN VLAN7. This enables non-rebooted UANs to continue to work
+   and allows for better easing into BICAN in CSM 1.2.
+
+### UAI migration
+
+Access to UAIs will be disrupted until CSM 1.2 upgrade completes. After the upgrade is completed, UAIs need to be restarted.
+
+### Decide on subnet ranges for new CAN/CHN
+
+Once you have decided whether to use the new CAN or to use CHN for user access, you must decide on the subnet range. Refer
+to [Customer Accessible Networks](../../operations/network/customer_accessible_networks/Customer_Accessible_Networks.md)
+for subnet ranges and defaults for CAN/CHN.
+
+### Preserving CMN subnet range
+
+It is vital that you preserve the subnet range for the pre-1.2 CAN that is now being renamed to CMN. Changing the subnet
```

Comment:
Rather than unintended consequences - which I love fwiw - this should likely say "is unsupported and will break the upgrade".  Or something like this.

Reply:
Thanks. Will make that change.